drop-boxes/register-nanopore-dropbox/register-nanopore.py

```diff
@@ -23,24 +23,24 @@
 from life.qbic.utils import NanoporeParser
 
 ######## Sample Tracking related import
-from life.qbic.sampletracking import SampleTracker
-from life.qbic.sampletracking import ServiceCredentials
-from java.net import URL
+#from life.qbic.sampletracking import SampleTracker
+#from life.qbic.sampletracking import ServiceCredentials
+#from java.net import URL
```

Comment:
Code/imports should not be commented out and committed. Better remove this codelines with a dedicated commit, such that this can be reverted later if needed.